examples/cifar/train_cifar_custom_loop.py

```diff
@@ -76,6 +77,17 @@ def main():
     optimizer.setup(model)
     optimizer.add_hook(chainer.optimizer.WeightDecay(5e-4))
 
+    if args.resume != '':
+        resume = args.resume
+        if os.path.exists(resume):
+            serializers.load_npz(os.path.join(resume, 'vgg.model'), model)
+            serializers.load_npz(os.path.join(resume, 'vgg.state'), optimizer)
+        else:
```

Comment:
Is this redundant?

Reply:
No, I think it's good (user friendly). Let's keep it as it. Preferably fix the MNIST example.